tracer/src/Datadog.Trace.Tools.Runner/Checks/Resources.cs

```diff
@@ -136,5 +141,11 @@ public static string WrongProfilerEnvironment(string environmentVariable, string
         }
 
         private static string EscapeOrNotSet(string? str) => str == null ? "not set" : $"'{str}'";
+
+        public static string TracerProgramFound(string tracerProgramName) => $"{tracerProgramName} found in the installed programs.";
+
+        public static string WrongTracerArchitecture(string tracerArchitecture) => $"Found {tracerArchitecture} installed but the current machine is X64 Bit, make sure to install the 64-bit tracer instead.";
+
+        public static string AppPoolCheckFindings(string appPool) => $"Checks not passing so far, update the configuration below which has been set on the {appPool} AppPool:";
```

Comment:
I would probably reword this one, maybe something like this?

```suggestion
        public static string AppPoolCheckFindings(string appPool) => $"Incorrect environment variable configuration found for {appPool} AppPool in <environmentVariables> configuration section: ";
```